test/k8sT/Policies.go

```diff
@@ -827,78 +736,157 @@ EOF`, k, v)
 		})
 	})
 
-	Context("KubernetesNetworkPolicy between server and client", func() {
+	Context("Namespaces policies", func() {
 
 		var (
-			policy = "allow-ns-b-via-namespace-selector"
+			err         error
+			ciliumPod   string
+			namespace   = "second"
+			appPods     map[string]string
+			appPodsNS   map[string]string
+			clusterIP   string
+			clusterIPNS string
```

Comment:
Rename this variable to be something like "app1SecondNSClusterIP" - the variable name in its current form is not evocative of what it actually represents, and makes the test hard to follow.